packages/cli/src/commands/logout/command.ts

```diff
@@ -0,0 +1,15 @@
+import { Command } from '../help';
+import { getPkgName } from '../../util/pkg-name';
+
+export const logoutCommand: Command = {
+  name: 'logout',
+  description: 'Logout of the currently authenticated user or team.',
```

Comment:
Something about this reads funny. How do you feel about slimming it down?

```suggestion
  description: 'Logout the current authenticated user or team.',
```